.github/workflows/build_wheels_linux.yml

```diff
@@ -55,6 +55,16 @@ on:
         description: "The key created when saving a cache and the key used to search for a cache."
         default: ""
         type: string
+      architecture:
+        description: Architecure to build for x64 for default Linux, or ARM64 for Linux aarch64 builds
+        required: false
+        type: string
+        default: x64
+      setup-miniconda:
```

Comment:
IIUC this is always going to be `false` for aarch64 and `true` otherwise. In that case, should we just make this an env var and set it accordingly instead of exposing as an arg to the caller.

Reply:
I think having input parameter is better, since its more flexible and more clear to the user , given that we already expose  setup-miniconda input parameter in setup binary builds: https://github.com/pytorch/test-infra/blob/main/.github/actions/setup-binary-builds/action.yml#L21